llpc/lower/llpcSpirvLowerConstImmediateStore.cpp

```diff
@@ -175,7 +175,7 @@ StoreInst *SpirvLowerConstImmediateStore::findSingleStore(AllocaInst *allocaInst
 // @param storeInst : The single constant store into the "alloca"
 void SpirvLowerConstImmediateStore::convertAllocaToReadOnlyGlobal(StoreInst *storeInst) {
   auto allocaInst = cast<AllocaInst>(storeInst->getPointerOperand());
-  auto globalType = allocaInst->getType()->getElementType();
+  auto globalType = allocaInst->getType()->getPointerElementType();
```

Comment:
Can we use `allocaInst->getAllocatedType()` here and above? That would be even more future proof.